src/Control/Retry.hs

```diff
@@ -157,7 +198,7 @@ fibonacciBackoff
     :: Int
     -- ^ Base delay in microseconds
     -> RetryPolicy
```

Comment:
We need to say `RetryPolicyM m` here right? Otherwise `fibonacciBackoff` cannot be used with `recovering`.